src/App.css

```diff
@@ -3,3 +3,305 @@
 *::after {
   box-sizing: border-box;
 }
+
+.Container {
+  background-color: rgb(249, 249, 249);
+}
+
+.NavBar {
+  background-color: rgb(255, 255, 255);
+}
+
+.ListGroup {
+  margin: 10px 0 10px 0;
+  border: solid !important;
+  border-width: 1px !important;
+  border-color: rgb(220, 220, 220) !important;
```

Comment:
- [ ] Overusing the !important rule is often considered a bad practice due to the clutter it can bring with one of CSS’s core mechanisms which is specificity. So I will suggest you limit its usage to maybe one. See this [Link](https://speckyboy.com/good-bad-css-practices/) for more information on CSS best practices

Reply:
Thanks, I reduce it to the minimum.